.travis.yml

```diff
@@ -29,57 +29,57 @@ addons:
       - zlib1g-dev
       - swig
 env:
-  - TESTS="export R2PM_GITDIR=${TRAVIS_BUILD_DIR} && r2pm -i armthumb"
-  - TESTS="export R2PM_GITDIR=${TRAVIS_BUILD_DIR} && r2pm -i baleful"
+  - TESTS="export R2PM_GITDIR='/home/travis/build/radare' && r2pm -i armthumb"
```

Comment:
Yeah I know but ${TRAVIS_BUILD_DIR} gives the incorrect path. Are you able to set a new env variable in the travis build system?